apps/next/src/pages/Approve/components/ActionDetails/btc/lib/btcDetailsUtils.ts

```diff
@@ -0,0 +1,72 @@
+import { TFunction } from 'react-i18next';
+import { DetailItemType, DetailSection } from '@avalabs/vm-module-types';
+
+import { BtcNetwork } from '@core/types';
+
+type MapFunction = (
+  network: BtcNetwork,
+  t: TFunction,
+) => (section: DetailSection, index: number) => DetailSection;
+
+// The funds recipients section for NextGen looks different than from the legacy one.
+// To keep things backwards-compatible, we map the recipients items here instead of
+// updating the Bitcoin Module.
+// FIXME: This should be changed (in sync with Core Mobile) once we fully migrate
```

Comment:
Please create a tech debt ticket for this.

Reply:
Yup -> [CP-11995](https://ava-labs.atlassian.net/browse/CP-11995)